tests/UI/campaigns/functional/BO/00_login/01_passwordReminder.js

```diff
@@ -0,0 +1,273 @@
+      await expect(pageTitle).to.contains(loginPage.pageTitle);
+    });
+
+    it('should send reset password mail', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'sendResetPasswordMailAndCheckSuccess', baseContext);
+      await loginPage.sendBOResetPasswordLink(page, createEmployeeData.email);
+
+      const successTextContent = await loginPage.checkSendResetPasswordLinkSuccess(page);
+      await expect(successTextContent).to.contains(resetPasswordSuccessText);
+    });
+
+    it('should check if reset password mail is in mailbox', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkIfResetPasswordMailIsInMailbox', baseContext);
+
+      await expect(newMail.subject).to.contains(resetPasswordMailSubject);
+    });
+  });
+
+  describe('Go to BO and delete previously created employee', async () => {
+    it('should login in BO', async function () {
+      await loginCommon.loginBO(this, page);
+    });
+
+    it('should go to Employees page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToEmployeesPageToDelete', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.advancedParametersLink,
+        dashboardPage.teamLink,
+      );
+
+      const pageTitle = await employeesPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(employeesPage.pageTitle);
+    });
+
+    it('should filter list by email', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterEmployeesToDelete', baseContext);
+
+      await employeesPage.filterEmployees(
+        page,
+        'input',
+        'email',
+        createEmployeeData.email,
+      );
+
+      const textEmail = await employeesPage.getTextColumnFromTable(page, 1, 'email');
+      await expect(textEmail).to.contains(createEmployeeData.email);
+    });
+
+    it('should delete employee', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'deleteEmployee', baseContext);
+
+      const textResult = await employeesPage.deleteEmployee(page, 1);
+      await expect(textResult).to.equal(employeesPage.successfulDeleteMessage);
+    });
+
+    it('should reset filter and check the number of employees', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'resetAfterDelete', baseContext);
+
+      const numberOfEmployeesAfterDelete = await employeesPage.resetAndGetNumberOfLines(page);
+      await expect(numberOfEmployeesAfterDelete).to.be.equal(numberOfEmployees);
+    });
+
+
+    it('should logout from BO', async function () {
+      await loginCommon.logoutBO(this, page);
+    });
+  });
+
+  describe('Go to BO and reset to default mail parameters', async () => {
+    it('should login in BO', async function () {
+      await loginCommon.loginBO(this, page);
+    });
+
+    it('should go to email setup page', async function () {
```

Comment:
```suggestion
  });

  describe('Go to BO and reset to default mail parameters', async () => {
    it('should go to email setup page', async function () {
```

We could avoid logout and login Here